pkg/billing/client.go

```diff
@@ -31,6 +31,12 @@ func (c Client) Hostname() string {
 	return c.hostname
 }
 
+type TraceID string
+
+func (c Client) GenerateTraceID() TraceID {
+	return TraceID(uuid.NewString())
```

Comment:
Or https://github.com/lithammer/shortuuid. It's faster to select strings without a dash.
But don't insist

Reply:
Done